### DISCUSSION TITLE
Does my blog homepage have pagination?

### DISCUSSION THREAD

**Discussion by ViSurya** [elegantframework/elegant-cli#351] [Q&A] [answered]
Firstly, thank you for creating this elegant framework!

I'm not very tech-savvy, and I want to migrate my blog from WordPress to Elegant Themes. Does the framework have a pagination feature? I have over 200 posts, and I need my visitors to be able to navigate through them easily.

**Comment by Bowens20832** [comment]
Hey @ViSurya, currently there is not any pagination built into the included theme.

However I can add it in no problem!

What did you have in mind?

**Comment by ViSurya** [comment]
I have a tech blog on wordpress with 250+ posts I want to migrate my blog
to elegant framwork

**Comment by Bowens20832** [comment]
Elegant should be able to handle 250+ blog posts with no problem!

Do you have a link to your current tech blog that I can check out?

**Comment by Bowens20832** [comment, marked as answer]
Hey @ViSurya 👋 

Pagination has been introduced in the latest version of Elegant, and I don't believe this is relevant any longer.

I am closing this discussion for now, but feel free to reopen it if you need.

Thank you again for supporting Elegant!
